addons/website_forum/controllers/main.py

```diff
@@ -153,10 +169,9 @@ def tags(self, forum, tag_char=None, **post):
         first_char_tag = forum.get_tags_first_char()
         first_char_list = [(t, t.lower()) for t in first_char_tag if t.isalnum()]
         first_char_list.insert(0, (_('All'), 'all'))
-        # get active first char tag
-        active_char_tag = first_char_list[1][1] if len(first_char_list) > 1 else 'all'
-        if tag_char:
-            active_char_tag = tag_char.lower()
+
+        active_char_tag = tag_char and tag_char.lower() or 'all'
```

Comment:
Old code was smarter ;)

If you take main forum of Odoo, you will see that you need several minute to display the all tags page or to see that the page is crashed... 
In most of the case, when you click on tag, you will click after on another letter.

So for performance issue, we decide to display the first letter of the first tag, instead of all tag.
Can you revert to old code plz

Reply:
Yes, indeed, I know that this expose a problem ([ know issues in the pad](https://www.odoo.com/web#id=2024197&action=333&active_id=587&model=project.task&view_type=form&menu_id=4720)).

Fp asked to improve performance here, the user should be able to see all tags without crashes (I imported odoo.com tags in my local instance to test).
Currently we render a `website_mail.follow` for each tag, can we do that "on hover" only using JS?

Reply:
> Currently we render a website_mail.follow for each tag, can we do that "on hover" only using JS?

not sure, just an idea

Reply:
no interest to display 5111 imo in one page...

We can use pager btw to catch this problem... but that continue to be an useless page ;)

Reply:
> no interest to display 5111 imo in one page...

^^ There should not be 5000 tags in the first place, but I got your point.
Let's assume you have a normal number of tags, having all of them in one page it's perfect, you can scroll and decide which one subscribe. Proposal:

```
tags < X
    'ALL' is default
tags > X
    'A' is default (not 0), hide the 'ALL' button
```

Also, we should avoid to let the user create duplicates using suggestion... eg in our forum we have: **v7**,   **v.7**,   **v7.0**,  **task**, **tasks**....

Reply:
If you find a way to educate the end user... I take it...
By default, all news users cannot create tag, maybe we should increase the karma limit :)

An task exists about merge of tag by Lemmatisation, but we don't really want it in Odoo.
PR 32386

Problem is to have website without moderator, or let user with small karma create it, or just don't suggest good tags to help them to find the existing similar tag... 
If we avoid the creation of wrong tag, we will not need to merge it;)

Reply:
Yes, to avoid unnecessary tags creation would be nice.
What about my proposal for the tags page? 

```
tags < X
    'ALL' is default
tags > X
    'A' is default (not 0), hide the 'ALL' button
```

Reply:
as well your solutions, as well a pager... everything is good for me ;) 

Be awrae, that the first selected is not really 0, but the first 'first letter' from all tag sorted.

So if you have tags: **2**0,  **v**12, **l**ucky, **5**0
you will have [2,5,L,V] => 2 as first selected tag

btw, does it take sense to use interger as tag ;) not really sure...